.github/workflows/README.md

```diff
@@ -11,27 +11,49 @@ The last part explains how to further develop this pipeline.
 
 ### Prerequisites
 Note: these instructions assume that the release is done from the `main` branch,
-and that the following worklows are present in the repository (cf. [below](#installation-of-the-release-pipeline)):
+and that the following workflows are present in the repository (cf. [below](#installation-of-the-release-pipeline)):
+- 'Bump version' (referencing [this workflow](https://github.com/MannLabs/alphashared/.github/workflows/bump_version.yml))
 - 'Create Draft Release' (referencing [this workflow](https://github.com/MannLabs/alphashared/.github/workflows/create_release.yml))
 - 'Publish on PyPi' (referencing [this workflow](https://github.com/MannLabs/alphashared/.github/workflows/publish_on_pypi.yml))
```

Comment:
This wasn't part of this commit but if we are updating the readme anyways we should probably remove the no longer functional links here.